### DISCUSSION TITLE
درخواست راهنمایی برای استفاده از ای پی آی ها جهت ساخت پنل مدیریت حساب کاربری

### DISCUSSION THREAD

**Discussion by soroushsafaar** [Gozargah/Marzban#1037] [Ideas]
امیدوارم حالتون خوب باشه.

من قصد دارم یک پنل مدیریت حساب برای کاربرانم بسازم و مایلم از ای‌پی‌آی‌های از پیش ساخته شده شما استفاده کنم. برای این منظور، چند سوال و درخواست دارم:

مستندات API: آیا می‌توانید مستندات کامل API‌های موجود را در اختیارم قرار دهید؟ این مستندات شامل توضیحات دقیق درباره هر endpoint، نحوه ارسال درخواست‌ها، پارامترهای ورودی و خروجی، و نمونه کدها باشد.

انواع API‌ها: چه نوع API‌هایی برای مدیریت حساب کاربران موجود است؟ به عنوان مثال، API‌هایی برای ایجاد، خواندن، بروزرسانی و حذف حساب‌های کاربری، مدیریت احراز هویت و مجوزها، و بررسی فعالیت‌های کاربری.

راهنمای شروع: آیا راهنمای شروع سریع یا نمونه پروژه‌ای دارید که بتوانم به عنوان مرجع استفاده کنم؟ این می‌تواند کمک زیادی به فهم بهتر نحوه استفاده از API‌ها کند.

نکات امنیتی: چه نکات امنیتی را باید در زمان استفاده از API‌ها در نظر بگیرم؟ آیا توصیه‌های خاصی برای مدیریت توکن‌های دسترسی، رمزنگاری داده‌ها، و جلوگیری از حملات رایج وجود دارد؟

از همکاری و راهنمایی شما سپاسگزارم. امیدوارم با کمک شما بتوانم بهترین تجربه کاربری را برای کاربرانم فراهم کنم.

با تشکر و احترام،

**Comment by alirezax5** [comment]
سلام
یک سرور با مرزبان بالا بیارید ، از .env کلید docs رو true کنید.
بعد ادرس پنلتون رو به صورت :
ip:8000/docs
بزنید مستندات میاد 
نمپنه کد هم داخل گیتهاب بزنید مرزبان پروژه های مرتبط میاد.

**Reply by soroushsafaar** [reply]
سپاس
